src/platform.cpp

```diff
@@ -2,6 +2,10 @@
 
 #if defined(_WIN32) || defined(__CYGWIN__)
 
+time_t timegm(struct tm *timeptr) {
+	return _mkgmtime(timeptr);
```

Comment:
link to doc please https://msdn.microsoft.com/en-us/library/2093ets1.aspx

and is it thread safe after all?